src/test/java/org/rulez/demokracia/pdengine/ComputedVoteTest.java

```diff
@@ -111,4 +112,13 @@ private BeatTable initBeatTable() {
 		computedVote.computeVote();
 		return computedVote.getBeatTable();
 	}
+	
+	@TestedBehaviour("vote result includes the votes cast with the secret cast vote identifier.")
+	@Test
+	public void vote_reulst_includes_the_votes_cast_with_the_secret_cast_vote_id() {
```

Comment:
no need to list the vote identifiers separately. They are part of the votes cast, which is part of the vote result.
testcasecount++1 for the effort

Reply:
So I don't have any other todo with it.

Reply:
not needed .... but still has to be fixed
(I asked about it)

Reply:
You just have to check that the vote results (which are meant to be in json) contain the vote result identifiers (in the votes cast list). The change gathering them separately should be rolled back.

Reply:
there's also a typo:
vote_re**uls**t_includes_ instead of
vote_re**sul**t_includes_

Reply:
there's also a typo:
vote_re**uls**t_includes_ instead of
vote_re**sul**t_includes_